content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -39,6 +39,11 @@ memory or CPU resources must be requested as well.
 A pod may consume multiple huge page sizes in a single pod spec. In this case it
 must use `medium: HugePages-<hugepagesize>` notation for all volume mounts.
 
+This Volume and VolumeMount are only necessary if your application
```

Comment:
I'd move this section to the bullet point list below and simplify to:

```
- emptyDir Volume/VolumeMount usage is not required if the pod consumes huge pages via `shmat`/`shmget` system calls or `mmap` with the `MAP_HUGETLB` flag.
```

Reply:
Hi @kirinrastogi , can you review at endocrimes' suggestion

Reply:
I did this but changed `emptyDir` to be `EmptyDir` to match the capitalization with the other bullet points, and used `with` instead of `via`.